CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# [<img src="https://avatars1.githubusercontent.com/u/3771091?v=3&s=30">](https://github.com/gonum) Contribution Guidelines
+#### Introduction
```

Comment:
A couple of post-LGTM suggestions.

How about removing this `#### Introduction` header, it's not really necessary and I don't like the jump from one `#` to four `####`.